backend/migrations.py

```
@@ -0,0 +1,56 @@
+    with engine.connect() as conn:
+        # Migration 001: Add phone column to users table
+        try:
+            if not column_exists('users', 'phone'):
+                logger.info("Running migration 001: Adding phone column to users table")
+                conn.execute(text("ALTER TABLE users ADD COLUMN phone VARCHAR"))
+                conn.commit()
+                logger.info("✓ Migration 001 completed: phone column added")
+            else:
+                logger.info("⊘ Migration 001 skipped: phone column already exists")
+        except OperationalError as e:
+            logger.error(f"✗ Migration 001 failed: {str(e)}")
```

Ruff (0.14.4)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)